backend/core/src/listings/views/config.ts

```diff
@@ -63,13 +63,15 @@ views.partnerList = {
   select: [
     "listings.id",
     "listings.name",
-    "listings.applicationDueDate",
+    "property.id",
+    ...getBaseAddressSelect(["buildingAddress"]),
```

Comment:
Do we need this address and the one on 73?

Reply:
on the partner site's listing grid we do display the address for the listing, so we do want to have that data